src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -290,6 +291,53 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
     }
 
     /*********************************************************************************************************
+    * @description Verifies HouseHold RecordType is not the default RecordType for the current User.
+    */
+    @TestVisible
+    private void verifyAccountDefaultRecordType() {
+        try {
+           
+            Id npspHouseholdRecordType = UTIL_CustomSettingsFacade.getOrgContactsSettings().npe01__HH_Account_RecordTypeID__c;
+            Id npspOneToOneRecordType = UTIL_CustomSettingsFacade.getOrgContactsSettings().npe01__One_to_One_RecordTypeID__c;
+            Map<Id,Schema.RecordTypeInfo> accountRecordTypeInfosById = Account.SObjectType.getDescribe().getRecordTypeInfosById();
+
```

Comment:
we have a class, UTIL_Describe, that caches describe info so that within the execution context we avoid multiple describe calls.  take a look at what's in that class, and consider whether you should leverage it instead, or add a getRecordTypeInfosById method to that class.